### PR TITLE
Update Vox Pupuli Containers

### PR DESCRIPTION
**Pull request by rwaffen**
- update Puppetserver to 7.17.0-v1.5.0
- update PuppetDB to 7.18.0-v1.5.0
- update Puppetboard to 5.4.0

### CODE REVIEW

**Comment by ldaneliukas** [comment]
Thanks @rwaffen! Could you bump the chart version and update the changelog/tests accordingly?

**Comment by rwaffen** [comment]
okay, will do.

**Comment by Xtigyro** [comment]
Hey @rwaffen - how is it going?

**Comment by rwaffen** [comment]
will get to it next friday. that's my foss day. have a reminder for it already

**Comment by Xtigyro** [comment]
@rwaffen Sweet - thank you!

**Comment by rwaffen** [comment]
how do you generate you changelog?

**Comment by rwaffen** [comment]
ah okay, should rebase 🤦

File: Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: puppetserver
-version: 9.3.4
-appVersion: 7.13.0
+version: 9.4.0
+appVersion: 7.17.0-v1.5.0
```
Comment (Xtigyro) [inline Chart.yaml]: `-v1.5.0` should be removed everywhere.

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: Chart.yaml (hunk 1 above)
Comment (rwaffen) [reply Chart.yaml]: not sure about that, because thats the container tag, but if you say so

File: README.md [hunk 2]
```diff
@@ -205,7 +205,7 @@ The following table lists the configurable parameters of the Puppetserver chart
 | `global.extraEnvSecret`| add extra environment variables to all containers from pre-existing secret |``|
 | `puppetserver.name` | puppetserver component label | `puppetserver`|
 | `puppetserver.image` | puppetserver image | `voxpupuli/container-puppetserver`|
-| `puppetserver.tag` | puppetserver img tag | `7.13.0`|
+| `puppetserver.tag` | puppetserver img tag | `7.17.0-v1.5.0`|
```
Comment (Xtigyro) [inline README.md:208]: Could we remove it from here too please?

File: README.md [hunk 3]
```diff
@@ -405,7 +405,7 @@ The following table lists the configurable parameters of the Puppetserver chart
 | `puppetdb.enabled` | puppetdb component enabled |`true`|
 | `puppetdb.name` | puppetdb component label | `puppetdb`|
 | `puppetdb.image` | puppetdb img | `voxpupuli/container-puppetdb`|
-| `puppetdb.tag` | puppetdb img tag | `7.14.0`|
+| `puppetdb.tag` | puppetdb img tag | `7.18.0-v1.5.0`|
```
Comment (Xtigyro) [inline README.md:408]: And here:

File: tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap [hunk 4]
```diff
@@ -50,7 +50,7 @@ manifest should match snapshot:
                 - name: CA_MASTERPORT
                   value: "8140"
               envFrom: null
-              image: ghcr.io/voxpupuli/container-puppetserver:7.13.0
+              image: ghcr.io/voxpupuli/container-puppetserver:7.17.0-v1.5.0
```
Comment (Xtigyro) [inline tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap:53]: And here:

File: tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap [hunk 5]
```diff
@@ -114,7 +114,7 @@ manifest should match snapshot:
                 - -c
               env: null
               envFrom: null
-              image: ghcr.io/voxpupuli/container-puppetserver:7.13.0
+              image: ghcr.io/voxpupuli/container-puppetserver:7.17.0-v1.5.0
```
Comment (Xtigyro) [inline tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap:117]: And here:

File: tests/__snapshot__/puppetserver-compilers.statefulset_test.yaml.snap [hunk 6]
```diff
@@ -53,7 +53,7 @@ manifest should match snapshot:
                 - name: CA_MASTERPORT
                   value: "8140"
               envFrom: null
-              image: ghcr.io/voxpupuli/container-puppetserver:7.13.0
+              image: ghcr.io/voxpupuli/container-puppetserver:7.17.0-v1.5.0
```
Comment (Xtigyro) [inline tests/__snapshot__/puppetserver-compilers.statefulset_test.yaml.snap:56]: And here:

File: tests/__snapshot__/puppetserver-compilers.statefulset_test.yaml.snap [hunk 7]
```diff
@@ -112,7 +112,7 @@ manifest should match snapshot:
                 - -c
               env: null
               envFrom: null
-              image: ghcr.io/voxpupuli/container-puppetserver:7.13.0
+              image: ghcr.io/voxpupuli/container-puppetserver:7.17.0-v1.5.0
```
Comment (Xtigyro) [inline tests/__snapshot__/puppetserver-compilers.statefulset_test.yaml.snap:115]: And here:

File: tests/__snapshot__/puppetserver-statefulset.compilers_test.yaml.snap [hunk 8]
```diff
@@ -53,7 +53,7 @@ manifest should match snapshot:
                 - name: CA_MASTERPORT
                   value: "8140"
               envFrom: null
-              image: ghcr.io/voxpupuli/container-puppetserver:7.13.0
+              image: ghcr.io/voxpupuli/container-puppetserver:7.17.0-v1.5.0
```
Comment (Xtigyro) [inline tests/__snapshot__/puppetserver-statefulset.compilers_test.yaml.snap:56]: And here:

File: tests/__snapshot__/puppetserver-statefulset.compilers_test.yaml.snap [hunk 9]
```diff
@@ -112,7 +112,7 @@ manifest should match snapshot:
                 - -c
               env: null
               envFrom: null
-              image: ghcr.io/voxpupuli/container-puppetserver:7.13.0
+              image: ghcr.io/voxpupuli/container-puppetserver:7.17.0-v1.5.0
```
Comment (Xtigyro) [inline tests/__snapshot__/puppetserver-statefulset.compilers_test.yaml.snap:115]: And here:

File: values.yaml [hunk 10]
```diff
@@ -63,7 +63,7 @@ global:
 puppetserver:
   name: puppetserver
   image: ghcr.io/voxpupuli/container-puppetserver
-  tag: 7.13.0
+  tag: 7.17.0-v1.5.0
```
Comment (Xtigyro) [inline values.yaml:66]: And here:

File: values.yaml [hunk 11]
```diff
@@ -750,7 +750,7 @@ puppetdb:
   enabled: true
   name: puppetdb
   image: ghcr.io/voxpupuli/container-puppetdb
-  tag: 7.14.0
+  tag: 7.18.0-v1.5.0
```
Comment (Xtigyro) [inline values.yaml:753]: And here:

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap (hunk 4 above)
Comment (rwaffen) [reply tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap:53]: how should this work? try pulling 7.17.0, this image tag does not exist.

Comment (Xtigyro) [reply tests/__snapshot__/puppetserver-compilers.deployment_test.yaml.snap:53]: Hmm, yeah, you're right - my bad!

**Review by Xtigyro — APPROVED** [review]

**Comment by Xtigyro** [comment]
@rwaffen Thank you once more!

**Pull request merged**